prow/gerrit/adapter/adapter.go

```diff
@@ -303,6 +303,7 @@ func (c *Controller) Sync() {
 		if len(changes) == 0 {
 			return
 		}
+		log.Infof("Processing %d changes for instance: %s", len(changes), instance)
```

Comment:
Suggest using log.WithFields() instead of Infof for structured logs and easier query(Exactly the same log message every time )